WireDomain/Sources/WireDomain/Repositories/User/UserRepository.swift

```diff
@@ -266,6 +278,26 @@ public final class UserRepository: UserRepositoryProtocol {
         }
     }
 
+    public func updateUserProperty(_ userProperty: UserProperty) async throws {
+        let selfUser = fetchSelfUser()
```

Comment:
This is only necessary for the read receipts case, so I would move it there.

Reply:
done